libcnb-test/tests/integration_test.rs

```diff
@@ -157,7 +161,7 @@ fn packaging_failure_missing_buildpack_toml() {
 // TODO: Fix the implementation to return the correct error message (that the buildpack.toml doesn't match the schema):
 // https://github.com/heroku/libcnb.rs/issues/708
 #[should_panic(
-    expected = "Could not package directory as buildpack! No buildpack with id `libcnb-test/invalid-buildpack-toml` exists in the workspace at"
+    expected = "Error packaging buildpack 'libcnb-test/invalid-buildpack-toml': Couldn't find a buildpack with ID 'libcnb-test/invalid-buildpack-toml' in the workspace at"
```

Comment:
Hmm, this isn't the error I was expecting to see for this test.  The buildpack.toml is invalid because it's missing the required `version` field so shouldn't we see a parse error like `Could not read buildpack.toml: <parse error details>`?

Reply:
Yeah - hence the TODO above it (it's another pre-existing bug)